ses/client.bal

```diff
@@ -0,0 +1,519 @@
+        return check handleHttpResponse(httpResponse);
+    }
+
+    # Updates a contact's preferences for a list. It is not necessary to specify all existing topic preferences in
+    # the TopicPreferences object, just the ones that need updating.
+    #
+    # + contactListName - The name of the contact list
+    # + emailAddress - The contact's email addres
+    # + contactUpdateRequest - The request payload to update the contact
+    # + return - An empty json on success else an error
+    @display {label: "Update Contact"}
+    remote isolated function updateContact(@display {label: "Contact List Name"} string contactListName,
+                                           @display {label: "Email Address"} string emailAddress, 
+                                           @display {label: "Request To Update Contact"}
+                                           ContactUpdateRequest contactUpdateRequest)
+                                           returns @display {label: "Contact Update Response"} json|error {
+        string requestUri = URI + "contact-lists/" + contactListName + "/contacts/" + emailAddress;
+        map<string> signedRequestHeaders = check getSignedRequestHeaders(self.host, self.accessKey, self.secretKey,
+                                                                         self.region, PUT, requestUri,
+                                                                         contactUpdateRequest.toString());    
+        http:Response httpResponse = check self.awsSesClient->put(requestUri, contactUpdateRequest.toString(),
+                                                                  signedRequestHeaders);                                                                   
+        return check handleHttpResponse(httpResponse);
+    }
+
+    # Returns a contact from a contact list.
+    #
+    # + contactListName - The name of the contact list
+    # + emailAddress - The contact's email addres
+    # + return - An aws.ses:Contact record on success else an error
+    @display {label: "Get Contact"}
+    remote isolated function getContact(@display {label: "Contact List Name"} string contactListName,
+                                        @display {label: "Email Address"} string emailAddress)
+                                        returns @display {label: "Contact"} Contact| error {
+        string requestUri = URI + "contact-lists/" + contactListName + "/contacts/"+ emailAddress;
+        map<string> signedRequestHeaders = check getSignedRequestHeaders(self.host, self.accessKey, self.secretKey,
+                                                                         self.region, GET, requestUri, EMPTY_STRING);
+        http:Response httpResponse = check self.awsSesClient->get(requestUri, signedRequestHeaders);
+        return (check handleHttpResponse(httpResponse)).cloneWithType(Contact);
```

Comment:
```suggestion
        return self.awsSesClient->get(requestUri, signedRequestHeaders, targetType = Contact);
```
Can't we do in this way?

Reply:
Resolved in an alternative way which is discussed in code review.